### DISCUSSION TITLE
Issue in viscousDrag post processing

### DISCUSSION THREAD

**Discussion by vanchisen** [Nek5000/nekRS#519] [General]
Dear NekRS,

I want to calculate the viscous force in flow past a circular cylinder.
I have used postProcessing::viscousDrag() following  /examples/ ktauChannel/ ci.inc..
 However, I get stuck at this snipped of the code, in particular, NekRS stop at **o_drag.CopyTo(drag)**.   
So I wonder if any one could please take a look at my mesh or input parameters ?  in case  they not compatible with the postProcessing functionality ?  
The Gmsh *.geo, input  *.par *.udf *.usr, *.oudf are located in following link,
https://github.com/vanchisen/NekRS_Examples  

Best Wishes,
Zhicheng Wang

**Comment by vanchisen** [comment]
Hi, I find a way to work around this issue, which is commenting out the @barrier in line 60 in /kernels/postProcessing/drag.okl 
Not sure if it gives me correct value. 

Best,

**Reply by stgeke** [reply]
This seems to be the correct fix.
